static/styles/zulip.css

```diff
@@ -1512,6 +1512,11 @@ blockquote p {
     width: 100%;
     margin-top: 6px;
 }
+@media (max-width: 775px) {
+    #below-compose-content {
+        text-align: right;
+    }
+}
```

Comment:
We've been trying to keep the @media sections grouped in a single section so I think it'd be better to move this to be part of the existing max-width 775 section (and similarly for the other blocks).